.pipelines/ci.yml

```diff
@@ -6,6 +6,9 @@ trigger:
   paths:
     exclude:
     - docs/*
+  tags:
+    include:
+    - v2*
```

Comment:
Why v2?

Reply:
because the tags will be date formatted: `v20210818.01` and I thought with `v2*` it gives some time until we have to update.